draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -612,13 +612,146 @@ array instead of retrieving the respective Type Metadata elsewhere as follows:
    Type Metadata document matches that of the `vct` in the Type Metadata document, and it MUST verify the integrity of the Type Metadata document as defined in
    (#document-integrity). The Consumer MUST NOT use the Type Metadata if no hash for integrity protection was provided.
 
+## Extending Type Metadata {#extending-type-metadata}
+
+A SD-JWT VC type can extend another type. The extended type is identified by the URI in
+the `extends` property. Consumers MUST retrieve and process
+Type Metadata for the extended type before processing the Type Metadata for the extending
+type.
+
+The extended type MAY itself extend another type. This can be used to create a
+chain or hierarchy of types. The security considerations described in
+(#circular-extends) apply in order to avoid problems with circular dependencies.
+
+## Schema Type Metadata {#schema-type-metadata}
+
+### Schema Definition {#schema-definition}
+
+Schemas for Verifiable Credentials are contained in the `schema` or retrieved via the `schema_uri` Type Metadata parameters (as defined in (#type-metadata-format)).
+A Schema MUST be represented by a JSON Schema document according to draft version 2020-12 [@JSON.SCHEMA.2020-12], or above.
+
+The Schema of a Verifiable Credential MUST include all properties that are required by this specification and MUST NOT change their cardinality.
```

Comment:
must not change their type either or anything else that might affect their syntactic validity or semantic intent

Reply:
Can a JSON schema change the type of a credential? on the other hand, it is a good point that technically, it is possible to assert a specific `vct` value in the JSON schema which would be weird as it would prevent other types to extend that type, right?

Reply:
I meant the data type where the "their" seemed to refer to any particular property. But what you said too.